.github/workflows/windows-vcpkg.yml

```diff
@@ -92,10 +87,15 @@ jobs:
         antares-deps-version: ${{steps.antares-deps-version.outputs.prop}}
         os: ${{matrix.os}}
 
-    - name: Install dependencies
-      run: |
-          python -m pip install --upgrade pip
-          pip3 install -r src/tests/examples/requirements.txt
+    - name: Recover pip dependencies from cache
+      uses: actions/setup-python@v4
```

Comment:
If python is not found, will there be an error and will the CI stop ?

Reply:
Yes